Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
+    money (6.13.6)
```

Comment:
### [money](https://rubymoney.github.io/money)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.13.4 → 6.13.6

[_[change-log](https://github.com/RubyMoney/money/blob/master/CHANGELOG.md), [source-code](https://github.com/RubyMoney/money/)_]





<details>
<summary>Commits</summary>

A change of **13** commits. See the full changes on [the compare page](https://github.com/RubyMoney/money/compare/v6.13.4...v6.13.6).

These are the first 10 commits:
- (29813d1) [Do not namespace specs with classes and modules (#873)](https://github.com/RubyMoney/money/commit/29813d1cad49678615a6f44f0eb488f9caae28b8)
- (4293055) [Update rubies on CI to latest versions (#874)](https://github.com/RubyMoney/money/commit/4293055abc1f5f16790cc90bc66ae270d1ed6532)
- (6be1cfb) [Raise warning on using Money.default_currency (#882)](https://github.com/RubyMoney/money/commit/6be1cfb10c49152d14b1efc314ba311f205c3145)
- (8754d9a) [Raise warning on using default Money.rounding_mode (#883)](https://github.com/RubyMoney/money/commit/8754d9ae614b8903525746589a6fed104d469344)
- (53f9a6e) [Add Second Ouguiya MRU 929 to currency iso file (#853)](https://github.com/RubyMoney/money/commit/53f9a6e2b0c2bbbbeb1419750389581c15a3b113)
- (3fdbbeb) [Change dist to trusty to support older rubies (#885)](https://github.com/RubyMoney/money/commit/3fdbbeb96bcae35fb27dcb604d15c35cde724bd9)
- (3ca8a3b) [Add symbol for UZS (#893)](https://github.com/RubyMoney/money/commit/3ca8a3b95a1230315b5b4e19982b3520a734ad11)
- (575fdcf) [Fix grammar in README.md (#899)](https://github.com/RubyMoney/money/commit/575fdcf1c4c420ea8cc27514da1088f9c6263056)
- (57c3da8) [Use monitor for recursive mutual exclusion in RatesStore::Me…](https://github.com/RubyMoney/money/commit/57c3da8675948b53a1605fc1a1eccbaa460d9599)
- (f5478c1) [Allow passing store as a string to Money::Bank::VariableExch…](https://github.com/RubyMoney/money/commit/f5478c1d4d4390281e742116fee559143ca52d70)


</details>